jib-gradle-plugin/src/integration-test/java/com/google/cloud/tools/jib/gradle/SingleProjectIntegrationTest.java

```diff
@@ -117,6 +121,12 @@ private static void assertDockerInspect(String imageReference)
                 + "            }"));
   }
 
+  static String assertDigestFile(Path digestPath) throws IOException, DigestException {
```

Comment:
Looking at this again, I think `readDigestFile()` sounds better than `assertDigestFile`.